recipes/cadishi/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "cadishi" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2e5e37416f90be2ae4bcd0bbb2d9ada1f7da1776256fef6efbaf6e5add8ed8d2
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - six
+    - future
+    - numpy
```

Comment:
Are you linking against numpy?  If so, this needs to be done with https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=numpy#building-against-numpy

Reply:
Yes, thanks for the hint. I am linking against NumPy correctly in the latest commit.